test/helpers.ts

```diff
@@ -279,3 +279,15 @@ export function prettyPrintPuppeteerAssertionError(message) {
 		assert.equal(match[1], match[2]);
 	}
 }
+
+export async function retryAsync<T>(fn: () => Promise<T>, maxAttempts: number = 3, interval: number = 1000): Promise<T> {
+	let attempts = 0;
+	while (attempts <= maxAttempts) {
+		try {
+			return await fn();
+		} catch (err) {
+			if (++attempts >= maxAttempts) throw err;
+			new Promise(resolve => setTimeout(resolve, interval));
```

Comment:
```suggestion
			await new Promise(resolve => setTimeout(resolve, interval));
```

Do we need an await here?

Reply:
Sorry!
https://github.com/sveltejs/svelte/pull/7076/commits/ebf5ff84f85517bf237e8d7e3a039b34ac3ed5c3

Reply:
NOTE: Added interval because there seemed to be cases where Chromium was failing to start continuously.
I'm not sure it will actually increase the success rate of tests, but I'd like to see how it goes.